enterprise/api/v1/serializers.py

```diff
@@ -214,3 +215,88 @@ def update(self, instance, validated_data):
     user = UserSerializer(read_only=True)
     enterprise_customer = EnterpriseCustomerSerializer(read_only=True)
     data_sharing_consent = UserDataSharingConsentAuditSerializer(many=True, read_only=True)
+
+
+class EnterpriseCatalogSerializer(serializers.Serializer):
```

Comment:
Also I see below that we are preventing state-changing requests -- perhaps this should be `EnterpriseCourseCatalogReadOnlySerializer`